static/files/forms-data.json

```
@@ -404,6 +404,61 @@
           ]
         }
       ]
+    },
+    "/solutions/open-source-security/cyber-resilience-act": {
+      "templatePath": "/solutions/open-source-security/cyber-resilience-act/index.html",
+      "isModal": true,
+      "modalId": "cra-contact-modal",
+      "formData": {
+        "title": "Get in touch with our experts on the Cyber Resilience Act (CRA) compliance",
+        "formId": "6051",
+        "returnUrl": "https://canonical.com/solutions/open-source-security/cyber-resilience-act/index.html#contact-form-success",
+        "product": ""
+      },
+      "fieldsets": [
+        {
+          "title": "Fill in this form and a member of our team will be in touch shortly.",
+          "id": "contact-form",
```

```suggestion
          "id": "contact-form",
          "noCommentsFromLead": true,
```
This makes sure it is appended to the payload correctly